tr.org.pardus.pen.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: tr.org.pardus.pen
+runtime: org.kde.Platform
+runtime-version: 6.7
+sdk: org.kde.Sdk
+command: pardus-pen
+finish-args:
+  - --socket=x11
+  - --device=dri
+modules:
+
+  - name: pardus-pen
+    buildsystem: meson
+    build-options:
+      env:
+        PATH: /app/bin:/usr/libexec:/usr/bin
+    config-opts:
+      - -Dqt=6
+      - --prefix=/app
+      - -Dscreenshot=false
+    sources:
+      - type: archive
+        url: https://github.com/pardus/pardus-pen/archive/refs/tags/debian/3.0.0.tar.gz
+        sha256: 155c423eb59a4cf282b0199f2af4137f796f53716090166d74085a0063a33d45
+
+  - name: glib-compile-schemas
```

Comment:
this should just be a post-install step in the main module. albeit I wonder why it is needed?

Reply:
done